lib/libzpool/taskq.c

```diff
@@ -213,6 +213,19 @@ taskq_wait_outstanding(taskq_t *tq, taskqid_t id)
 	taskq_wait(tq);
 }
 
+int
```

Comment:
comment related to function

Reply:
Sir I think this is not latest code. ....I have renamed it to taskq_check_active_ios(). Please do let me know if you meant else

Reply:
just few lines of comment explaining the i/p and o/p of this function